### PR TITLE
Multi-line threedotting issues in modals

### PR DESCRIPTION
**Pull request by nicolaasmatthijs**
screen shot 2015-03-19 at 10 11 24

### CODE REVIEW

**Comment by nicolaasmatthijs** [comment]
Attached a fix. Over to @mrvisser for review

**Comment by mrvisser** [comment]
@nicolaasmatthijs this should happen for `foldershared.js` as well I think

**Comment by nicolaasmatthijs** [comment]
That escalated quickly. Back to @mrvisser

File: node_modules/oae-core/sharedwith/tests/sharedwith.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
```
Comment (mrvisser) [inline node_modules/oae-core/sharedwith/tests/sharedwith.js]: `2015` :)

**Comment by nicolaasmatthijs** [comment]
Followed up on comment. Back to @mrvisser

File: node_modules/oae-core/sharedwith/js/sharedwith.js [hunk 2]
```diff
@@ -57,24 +58,27 @@ define(['jquery', 'oae.core'], function($, oae) {
         };
 
         /**
-         * Initialize the discussionshared modal dialog
+         * Initialize the sharedwith modal dialog
          */
-        var setUpDiscussionShared = function() {
-            $(document).on('click', '.oae-trigger-discussionshared', function(ev, data) {
+        var setUpsharedwith = function() {
```
Comment (mrvisser) [inline node_modules/oae-core/sharedwith/js/sharedwith.js]: `setUpSharedWith`, I'll fix

File: node_modules/oae-core/sharedwith/js/sharedwith.js [hunk 3]
```diff
@@ -26,21 +26,22 @@ define(['jquery', 'oae.core'], function($, oae) {
         /**
          * Initialize a new infinite scroll container that fetches the members
          *
-         * @param  {Discussion}    discussionProfile    Discussion for which the members should be rendered
+         * @param  {Content|Discussion|Folder}    contextProfile    Entity for which the members should be rendered
```
Comment (mrvisser) [inline node_modules/oae-core/sharedwith/js/sharedwith.js]: We don't really have a reliable process to come back and update this for each resource type that gets added in the future. We should probably just specify `Object` and indicate it accepts any resource model with a `resourceType`, `id` and a members list

Comment (mrvisser) [inline node_modules/oae-core/sharedwith/js/sharedwith.js]: I'll fix this too

**Pull request merged**